docs/website/learnmore/catalog/brooklyn-node.html

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
brooklyn-node refers to the specific `brooklyn.entity.brooklynnode.BrooklynNode` entity. `item.html` as in `Catalog item` would be more appropriate.